lib/siwapp/invoices_context.ex

```diff
@@ -0,0 +1,86 @@
+defmodule Siwapp.InvoicesContext do
```

Comment:
why call it "InvoicesContext"? I think Invoices would be better. 
Besides, the methods are kind of repetitive, they will be called like:
Invoices.list_invoices
Invoices.create_invoice
...
It would be better like this:
Invoices.list
Invoices.create
...

Reply:
I have called it InvoicesContext because there is already a module called invoices in which the Schema is defined (in another folder) but I thought that there were two modules with the same name could be confusing